web-app/src/components/BookingModal/container.js

```diff
@@ -83,6 +83,7 @@ const Container = Wrapped =>
         halfDay: isHalfday,
         eventId: eventId,
         startDate: start.format(this.dateFormat),
+        message: updateHolidayReason,
```

Comment:
I think this would be better named as "updateMessage" saying it's the terminology we use in the request API.